src/lib/Icon.tsx

```diff
@@ -10,30 +10,27 @@ import {
 } from "@fluentui/react-icons-northstar";
 
 export interface IIconProps {
-  icon?: string;
+  icon?: string | React.ReactElement;
```

Comment:
This change will also need to be propagated to the the TAction interface to allow the use it in the List, Toolbar, Board components.

Reply:
Good call.